instrumentation/resque/lib/opentelemetry/instrumentation/resque/instrumentation.rb

```diff
@@ -18,8 +18,38 @@ class Instrumentation < OpenTelemetry::Instrumentation::Base
           defined?(::Resque)
         end
 
-        option :span_naming,       default: :queue, validate: %I[job_class queue]
-        option :propagation_style, default: :link,  validate: %i[link child none]
+        ## Supported configuration keys for the install config hash:
+        #
+        # force_flush: controls if spans are forcibly flushed upon job completion
+        #   - :ask_the_job (default) - if `Resque::Worker#fork_per_job?` is set,
+        #     all completed spans will be synchronously flushed at the end of a
+        #     job's execution
+        #   - :always - all completed spans will be synchronously flushed at the
+        #     end of a job's execution
+        #   - :never - the job will not intervene with the processing of spans
+        #
+        # when `true`, all completed spans will be synchronously flushed
```

Comment:
I ❤️  the documentation addition!

But I'm confused: when _what_ is `true`? `force_flush`? Is `true` an acceptable value for the `force_flush` config option? That seems to conflict with the list of symbols above and the `validate: %I[ask_the_job always never]` validations below.

Reply:
Good catch @robbkidd! This is a derp. This section should be deleted. Commit incoming.

Reply:
Peace has been restored [4855a4e](https://github.com/open-telemetry/opentelemetry-ruby-contrib/pull/117/commits/4855a4e3dcc2475fc4df7c761b5500f8286e7c67)

Reply:
Let me know what you think now @robbkidd, I'll refrain from responding to your other comment. Apologies for the confusion.